test/arcade/patch/agent/cell/PatchCellRandomTest.java

```diff
@@ -64,6 +64,37 @@ public static void setupMocks() {
         doReturn(0).when(parametersMock).getInt(any(String.class));
     }
 
+    @Test
```

Comment:
I think these tests should be in `PatchCellTest` and not `PatchCellRandomTest`

Reply:
create `PatchCellTest.java` and move tests to this file